target_chains/cosmwasm/contracts/pyth/src/state.rs

```diff
@@ -65,15 +65,6 @@ pub struct ConfigInfo {
     pub fee: Coin,
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug, Default, PartialEq, Eq, JsonSchema)]
-#[serde(rename_all = "snake_case")]
-pub struct PriceInfo {
```

Comment:
you deleted this because we don't use any of the info here for anything anymore?

Reply:
yeah.